src/features/infinite-scroll/js/infinite-scroll.js

```diff
@@ -368,6 +368,11 @@
 
           newVisibleRows = grid.getVisibleRowCount();
 
+          var canvasHeight = rowHeight * newVisibleRows;
+          if (grid.infiniteScroll.scrollDown && (viewportHeight > canvasHeight)) {
+            grid.api.infiniteScroll.raise.needLoadMoreData();
+          }
+
```

Comment:
Can you add a comment explicitly describing what is going on here and why. Without the context of the commit message future developers will have no idea why this chunk of code is here.